web/styles/left_sidebar.css

```diff
@@ -540,6 +540,72 @@ li.topic-list-item:hover .topic-sidebar-menu-icon {
     color: hsl(0deg 0% 53%);
 }
 
+.top-left-sidebar-menu-icon {
+    display: none;
+}
+
+.icon_layout {
+    padding: 10px 0 5px;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+
+    .top_left_texts {
+        display: none;
+    }
+
+    .filter-icon i {
+        padding-right: 0 !important;
+    }
+
+    .unread_count {
+        position: absolute;
+        bottom: 1rem;
+        left: 1.2rem;
+    }
+
+    .top_left_starred_messages .unread_count,
+    .top_left_drafts .unread_count {
+        background-color: hsl(0, 0.55%, 71.18%) !important;
+        border: 0 !important;
+        color: hsl(0, 0%, 100%) !important;
```

Comment:
Why do these all need `!important`? We generally try to use selector precedence where possible to ensure the right CSS applies to something.

Reply:
Removed `!important` and used the right selector precedence to ensure the right CSS applies.